Makefile

```diff
@@ -7,7 +7,7 @@ SCRIPTS_DIR := $(ROOT_DIR)/scripts
 KUDO_TOOLS_DIR := $(ROOT_DIR)/shared
 SPARK_OPERATOR_DIR := $(ROOT_DIR)/spark-on-k8s-operator
 
-export KONVOY_VERSION ?= v1.1.5
+export KONVOY_VERSION ?= v1.2.4
```

Comment:
out of curiosity, why not 1.3.0?

Reply:
I used the default version, specified here: https://github.com/mesosphere/data-services-kudo/blob/ae0d6d4e4af5e24575841fcf9615ab4f14d93f4c/vendor/konvoy.sh#L5
But I think we can bump it to `1.3.0`, let's see how it goes